containers/src/Data/Sequence/Internal.hs

```diff
@@ -1707,7 +1707,8 @@ replicateA n x
   | otherwise   = error "replicateA takes a nonnegative integer argument"
 {-# SPECIALIZE replicateA :: Int -> State a b -> State a (Seq b) #-}
 
--- | 'replicateM' is a sequence counterpart of 'Control.Monad.replicateM'.
+-- | 'replicateM' is the @Seq@ counterpart of
+-- @Control.Monad.@'Control.Monad.replicateM'.
```

Comment:
```suggestion
-- @"Control.Monad".'Control.Monad.replicateM'@.
```

I'm pretty sure that's how we usually do it, and it feels better.

Reply:
OK, as you prefer.

Reply:
(FWIW I think it's less clear to link to the `Control.Monad` module *as well*. It distracts from point of the link, which is just to `replicateM`.)

Reply:
Ah, in that case drop the double quotes, but I still want the ampersand shifted to the end.

Reply:
Makes sense. Done.